src/main/resources/webapp/js/controllers.js

```diff
@@ -1240,7 +1274,7 @@
             return $scope.form
                 && $scope.form.dto
                 && $scope.form.dto.plannedDate
```

Comment:
I think it should be && (actualDate || (plannedDate && startTime))

Reply:
Yes you are right. I forgot to change it. Thanks!